app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Nit: Can you update this comment to only include the modules that might be relevant for future work with a short blurb about them? For example omniauthable isn't too relevant for what we're trying to build